cliquet/events.py

```diff
@@ -0,0 +1,15 @@
+from cliquet.utils import strip_uri_prefix
+
+
+class CollectionChanged(object):
+    """Triggered when a collection is changed.
+    """
+    def __init__(self, action, collection, request):
+        self.request = request
+
+        self.payload = {'collection_id': collection.record_id,
+                        'timestamp': collection.timestamp,
+                        'action': action,
+                        'uri': strip_uri_prefix(request.path),
+                        'user_id': request.unauthenticated_userid,
+                        'bucket_id': request.matchdict.get('bucket_id')}
```

Comment:
resource.name <-- what is the resource object in that case ? 

cliquet.collection.Collection and cliquet.resource.Resource  don't have a name attribute